### DISCUSSION TITLE
[Detector Support]:  Check the stream used for detection

### DISCUSSION THREAD

**Discussion by jerome71** [blakeblackshear/frigate#13671] [Detector Support] [answered]
### Describe the problem you are having

I am experimenting with Frigate.
Is there a way of checking the actual video stream used for detection?
That is only for testing purposes, do it's OK if it introduces additional CPU load.

I could not find how to do it, or record an identical stream with the same FFMPEG settings.

Thanks in advance.

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
detect:
      enabled: true 
      width: 1280
      height:
      fps: 5
```

### docker-compose file or Docker CLI command

```yaml
Not relevant here IMHO
```

### Relevant Frigate log output

```shell
Not relevant here IMHO
```

### Relevant go2rtc log output

```shell
Not relevant here IMHO
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

<img width="1840" alt="Capture d’écran 2024-09-11 à 09 15 37" src="https://github.com/user-attachments/assets/025cf1fa-c049-406e-b4f5-a57e425cbf2d">

**Comment by hawkeye217** [comment]
I'm not sure I understand your question. What do you mean "check the video stream"? 

You can look at Frigate's debug view for your camera (Settings --> Debug), which displays the stream being used for detection.

**Reply by jerome71** [reply]
I mean visualize it, and cheick its properties.
In other words, I'd like to see what the CPU sees for detection, and the TPU for classification.

At the moment, I can only do that with the source, before it gets compressed accodiing to the detect prperties.

**Reply by hawkeye217** [reply, marked as answer]
The Debug view sounds like what you're looking for.

**Reply by jerome71** [reply]
Great. Thanks. Exactly what I was lonokinf for :-)

**Reply by hawkeye217** [reply]
Great! Happy to help.
